src/com/aca/raf/management/PenVideoRecorder.java

```diff
@@ -0,0 +1,20 @@
+package com.aca.raf.management;
+
+public class PenVideoRecorder extends Gadget implements VideoRecorder{
+    private String writingColor;
+
+    public PenVideoRecorder(int duretion, int price, int count, String writingColor) {
```

Comment:
you allow put "BLACK", "GREEN" or  "DRUGS" values as a writing color Raf jan, restrict that case use Enums or put here validations